src/Components/Alert/Hooks/useAlertContext.tsx

```diff
@@ -163,23 +177,37 @@ export const AlertProvider: FC<AlertProviderProps> = ({
   initialCriteria,
   currentArtworkID,
 }) => {
-  const { relayEnvironment } = useSystemContext()
+  const { createdAlert } = useAlertTracking()
+  const { showAuthDialog } = useAuthDialog()
+  const { value, clearValue } = useAuthIntent()
+  const { submitMutation } = useCreateAlert()
+  const { isLoggedIn, relayEnvironment } = useSystemContext()
 
   const initialState = {
-    settings: DEFAULT_STATE.settings,
-    criteria:
-      getAllowedSearchCriteria(initialCriteria as SearchCriteriaAttributes) ??
-      DEFAULT_STATE.criteria,
+    settings: {
+      name: "",
+      details: "",
+      push: true,
+      email: true,
+    },
+    criteria: getAllowedSearchCriteria(
+      initialCriteria as SearchCriteriaAttributes
+    ),
     currentArtworkID,
     preview: null,
+    visible: false,
   }
-  const basis = useRef<State>(initialState)
   const [current, setCurrent] = useState<
     "ALERT_DETAILS" | "ALERT_FILTERS" | "ALERT_CONFIRMATION"
   >("ALERT_DETAILS")
 
-  const { submitMutation } = useCreateAlert()
-  const { createdAlert } = useAlertTracking()
+  useEffect(() => {
+    const criteria =
+      getAllowedSearchCriteria(initialCriteria as SearchCriteriaAttributes) ??
```

Comment:
As this is equivalent to the now forbidden non-null assertion (https://github.com/artsy/force/pull/13083), we should probably remove the assertion. For example, like this:

```suggestion
      getAllowedSearchCriteria(initialCriteria || {}) ??
```